lib/libc/minimal/include/stdlib.h

```diff
@@ -16,13 +16,27 @@
 extern "C" {
 #endif
 
+struct mallinfo2 {
+	size_t arena;    /* total space allocated from system */
+	size_t ordblks;  /* number of non-inuse chunks */
+	size_t smblks;   /* unused -- always zero */
+	size_t hblks;    /* number of mmapped regions */
+	size_t hblkhd;   /* total space in mmapped regions */
+	size_t usmblks;  /* unused -- always zero */
+	size_t fsmblks;  /* unused -- always zero */
+	size_t uordblks; /* total allocated space */
+	size_t fordblks; /* total non-inuse space */
+	size_t keepcost; /* top-most, releasable (via malloc_trim) space */
+};
+
 unsigned long strtoul(const char *nptr, char **endptr, int base);
 long strtol(const char *nptr, char **endptr, int base);
 unsigned long long strtoull(const char *nptr, char **endptr, int base);
 long long strtoll(const char *nptr, char **endptr, int base);
 int atoi(const char *s);
 
 void *malloc(size_t size);
+struct mallinfo2 mallinfo2(void);
```

Comment:
Same - please put this into mallinfo.h

See https://man7.org/linux/man-pages/man3/mallinfo.3.html

Reply:
I wanted to put this in `malloc.h` as per standard, but I see that malloc is implemented as part of `stdlib`.

Reply:
Again, regardless of where `malloc()` is _declared_, all declarations related to `mallinfo()` should go into `malloc.h`, per the manual page.

That's a kind of fundamental thing. Otherwise, if someone were reading the man page about how to use `mallinfo2()`, and they used `#include <malloc.h>`, and then ran their build job, there would be no function prototype declared and the compiler would fail.

```cpp
#include <stdint.h>

/* no such file or directory? */
#include <malloc.h>

int main() {
  /* undefined reference to mallinfo2 */
  return (uintptr_t) mallinfo2;
}
```

Reply:
@krish2718 - can you please address this comment?